File: Easy/Java/2697. Lexicographically Smallest Palindrome.java
class Solution {
    public String makeSmallestPalindrome(String s) {
        char[] palin = s.toCharArray();
        int n = s.length();

        for(int i = 0; i < n / 2; i++){
            int j = n - i - 1;
            if(palin[i] != palin[j]){
                if(palin[i] < palin[j])
                    palin[j] = palin[i];
                palin[i] = palin[j];
            }
        }

        return new String(palin);
    }
}

File: Easy/Python/2697. Lexicographically Smallest Palindrome.py
class Solution(object):
    def makeSmallestPalindrome(self, s):
        palin = list(s)
        n = len(s)

        for i in range(len(s) / 2):
            j = n - i - 1
            if palin[i] != palin[j]:
                if palin[i] < palin[j]:
                    palin[j] = palin[i]
                palin[i] = palin[j]

        return ''.join(palin)
